examples/flutter_gallery/lib/gallery/demos.dart

```diff
@@ -556,21 +556,16 @@ List<GalleryDemo> _buildGalleryDemos() {
     ),
   ];
 
-  // Keep Pesto around for its regression test value. It is not included
-  // in (release builds) the performance tests.
-  assert(() {
-    galleryDemos.insert(0,
-      GalleryDemo(
-        title: 'Pesto',
-        subtitle: 'Simple recipe browser',
-        icon: Icons.adjust,
-        category: _kDemos,
-        routeName: PestoDemo.routeName,
-        buildRoute: (BuildContext context) => const PestoDemo(),
-      ),
-    );
-    return true;
-  }());
+  galleryDemos.insert(0,
```

Comment:
This should be included in the galleryDemos list (after the '2D Transformations' demo), rather than being inserted here. There's no longer anything special about this demo.

Reply:
Done.